README.md

```diff
@@ -477,6 +477,12 @@ All statements above will return false if used with `===`
 
 🔗 [**Read More: Refactoring!**](/sections/testingandquality/refactoring.md)
 
+## ![✔] 4.10 Run tests in parallel
```

Comment:
Add the WHY - "Run tests in parallel for faster feedback loop"

(feedback loop is a term from the testing world)

#just-an-idea